################################################################################
SHORT DESCRIPTION: 
################################################################################
Some miscellaneous fixes to make QEMU behave in OpenXT.

################################################################################
LONG DESCRIPTION: 
################################################################################
Fixes:
 - Don't propagate error when no VGA BIOS binary is found. This will cause
the VGA device creation to fail.

################################################################################
CHANGELOG 
################################################################################
Author: Ross Philipson, philipsonr@ainfosec.com, 10/17/2016

################################################################################
REMOVAL 
################################################################################

################################################################################
UPSTREAM PLAN
################################################################################
No

################################################################################
INTERNAL DEPENDENCIES 
################################################################################

################################################################################
PATCHES 
################################################################################
Index: qemu-2.6.2/hw/pci/pci.c
===================================================================
--- qemu-2.6.2.orig/hw/pci/pci.c
+++ qemu-2.6.2/hw/pci/pci.c
@@ -2082,11 +2082,12 @@ static void pci_add_option_rom(PCIDevice
 
     size = get_image_size(path);
     if (size < 0) {
-        error_setg(errp, "failed to find romfile \"%s\"", pdev->romfile);
+        /* Do not propagate the error and cause device failure */
+        /* error_setg(errp, "failed to find romfile \"%s\"", pdev->romfile); */
         g_free(path);
         return;
     } else if (size == 0) {
-        error_setg(errp, "romfile \"%s\" is empty", pdev->romfile);
+        /*error_setg(errp, "romfile \"%s\" is empty", pdev->romfile); */
         g_free(path);
         return;
     }
